infra/gcp/infra.yaml

```diff
@@ -335,6 +335,7 @@ infra:
       k8s-staging-csi-secrets-store:
       k8s-staging-descheduler:
       k8s-staging-dns:
+      k8s-staging-dra-driver-cpu:
```

Comment:
I think this is for the legacy registries migrated from GCR that were created by the bash script and now you do like:
https://github.com/kubernetes/k8s.io/pull/7824

which can be auto-provisioned via CI

Reply:
Updated. Maybe [this section](https://github.com/kubernetes/k8s.io/tree/d4972985953ca48579ec588ce68c9f00801499b9/registry.k8s.io#creating-staging-repos) needs an update ?